docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/activation/identity-16.rst

```diff
@@ -0,0 +1,48 @@
+Identity
+========
+
+
+.. meta::
+  :description: Learn about Identity-16 - a simple operation that forwards the input to the output.
+
+**Versioned name**: *Identity-16*
+
+**Category**: *Activation*
```

Comment:
Activation makes sense, but wouldn't it fit better "Infrastructure" category?

Reply:
I was even considering 'internal' myself, I am not sure

Reply:
"Internal" is a category of OpenVINO operations which are not a part of public API, so it wouldn't be the right choice. "Infrastracture" is also an option, but "Identity" also called "Linear" is described as a type of activation function in many articles, and this is aligned with detailed op description below. As the category here doesn't affect the op logic or compatibility, in my opinion it's ok leave it as is (but always can be moved).